apiserver/observer/observer.go

```diff
@@ -97,22 +105,37 @@ func (m *Multiplexer) Login(entity names.Tag, model names.ModelTag, fromControll
 func (m *Multiplexer) RPCObserver() rpc.Observer {
 	rpcObservers := make([]rpc.Observer, len(m.observers))
 	for i, o := range m.observers {
-		rpcObservers[i] = o.RPCObserver()
+		observer := o.RPCObserver()
+		if observer == nil {
+			continue
+		}
+
+		rpcObservers[i] = observer
```

Comment:
This is actually not a change in behaviour, since we will still increment the counter, leaving a nil in the slice

Reply:
Ah, yes, I'll fix.